pandas/core/groupby/groupby.py

```diff
@@ -2236,7 +2236,7 @@ def _get_cythonized_result(
         if result_is_index and aggregate:
             raise ValueError("'result_is_index' and 'aggregate' cannot both be True!")
         if post_processing:
-            if not callable(pre_processing):
+            if not callable(post_processing):
```

Comment:
do we have a test that actually hits this?

Reply:
#34372 changes the path for GroupBy.std so that this is hit by std tests.